inc/Engine/Optimization/RUCSS/Database/Tables/UsedCSS.php

```diff
@@ -28,8 +28,7 @@ class UsedCSS extends Table {
 	 *
 	 * @var int
 	 */
-	protected $version = 20220920;
-
+	protected $version = 20220926;
```

Comment:
```suggestion
	protected $version = 20221104;
```

this need to be changed to be the last one